docs/book/src/topics/windows.md

```diff
@@ -6,7 +6,7 @@ CAPZ enables you to create Windows Kubernetes clusters on Microsoft Azure. We re
 
 ### Using Containerd for Windows Clusters
 
-To deploy a cluster using Windows, use the [Windows Containerd flavor template](../../../../templates/cluster-template-machinepool-windows-containerd.yaml).
+To deploy a cluster using Windows, use the [Windows flavor template](../../../../templates/cluster-template-machinepool-windows.yaml).
```

Comment:
We can also drop the docs section below named `### Using Docker EE and dockershim for Windows Clusters`

Reply:
done